adoc/TRD-SLES-SAP-HA-automation-quickstart-cloud.adoc

```diff
@@ -158,33 +161,54 @@ endif::[]
 
 == Preparing the SAP HANA media
 
-The SAP HANA Media needs to be made available so it can be accessed during the deployment.
-The SUSE SAP Automation project allows for three methods for presenting the SAP media:
+With the correct entitlement, SAP HANA media can be downloaded from the SAP Web site at https://support.sap.com/en/my-support/software-downloads.html. The SAP Media needs to be made available so it can be accessed during the deployment.
```

Comment:
Not really sure if we should write hard-coded URLs from other companies into our document